Makefile

```diff
@@ -85,14 +85,19 @@ go-mod: ## Cleanup and verify go modules
 default-profiles: ## Generate the default profiles
 	$(GO) run ./profiles
 
+.PHONY: deployments
+deployments: ## Generate the deployment files with kustomize
+	kustomize build --reorder=none deploy/base -o deploy/operator.yaml
+	kustomize build --reorder=none deploy/overlays/namespaced -o deploy/namespace-operator.yaml
```

Comment:
I guess this reorder flag applies to the resources and not the fields… :thinking:

Reply:
I am fine moving ahead as-is, then circling back to clean up, what do you think?

Reply:
I agree, thank you! :)